.github/workflows/deploy-docs.yml

```
@@ -0,0 +1,74 @@
+    branches: [main, rewire-content-to-new-docs]
+    paths:
+      - 'docs/**'
+      - 'packages/*/src/**'
+      - 'packages/*/package.json'
+      - '.docs/**'
+      - '.github/workflows/deploy-docs.yml'
+  pull_request:
+    branches: [main]
+    paths:
+      - 'docs/**'
+      - 'packages/*/src/**' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)